src/hardware/memory.cpp

```diff
@@ -57,7 +57,11 @@ static struct MemoryBlock {
 	} a20;
 } memory;
 
-HostPt MemBase;
+#ifndef PAGESIZE
+#define PAGESIZE 4096
+#endif
+
```

Comment:
Maybe `HOST_PAGESIZE`, to differentiate from other `PAGESIZE`s?

Reply:
I was keeping consistency with the other files that need the host pagesize, e.g. `core_dynrec.cpp`